ghost/core/test/e2e-webhooks/members.test.js

```diff
@@ -121,4 +121,55 @@ describe('member.* events', function () {
                 }
             });
     });
+
+    it('member.edited event is triggered', async function () {
+        const webhookURL = 'https://test-webhook-receiver.com/member-edited/';
+        await webhookMockReceiver.mock(webhookURL);
+        await fixtureManager.insertWebhook({
+            event: 'member.edited',
+            url: webhookURL
+        });
+
+        const res = await adminAPIAgent
+            .post('members/')
+            .body({
+                members: [{
+                    name: 'Test Member3',
+                    email: 'testemail3@example.com',
+                    note: 'test note3'
+                }]
+            })
+            .expectStatus(201);
+        
+        const id = res.body.members[0].id;
+
+        const updatedMember = {...res.body.members[0]};
+        updatedMember.name = 'Ghost';
+        updatedMember.email = 'ghost@example.com';
+        updatedMember.note = 'ghost note';
```

Comment:
Not something to change here necessarily, just small piece of feedback to improve future contributions. It's generally a more readable and maintainable solution when tests can achieve the needed state in as few steps as possible (unless it's some hacky shortcut that is vague). In this case, just editing the name would have triggered the 'member.edited' event. Less code - better code 😊